syntax = "proto3";

enum MessageType
{
  TEMPERATURE = 0;
}

message TemperatureMessage
{
    MessageType m_type = 1;
    float m_temperature = 2;
    int64 m_timeStamp = 3;
    uint64 m_deviceId = 4;
    uint64 m_sensorId = 5;
}